src/Worker/Delivery.php

```diff
@@ -309,6 +309,10 @@ private static function deliverDFRN($cmd, $contact, $owner, $items, $target_item
 		} else {
 			// The message could not be delivered. We mark the contact as "dead"
 			Contact::markForArchival($contact);
```

Comment:
Is this still relevant when we try the fallback?

Reply:
It would be reset when the transmission wouldn't fail.